src/models/hotspot_status.rs

```diff
@@ -0,0 +1,11 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct Status {
```

Comment:
put the status definition in the hotspot model file

Reply:
Noted with thanks!